src/index.ts

```diff
@@ -1,25 +1,30 @@
 import { type AddressInfo } from 'net'
 import { defineNuxtModule } from '@nuxt/kit'
-import notbundle, { type Configuration } from 'notbundle'
+import {
+  type Configuration,
+  build,
+  watch,
+  startup,
+} from 'vite-electron-plugin'
 
 export interface ElectronOptions extends Configuration { }
 
 export default defineNuxtModule<ElectronOptions>({
   setup(options, nuxt) {
     const isProduction = process.env.NODE_ENV === 'production'
 
-    options.output ??= 'dist-electron'
+    options.outDir ??= 'dist-electron'
     options.watch ??= {}
     options.plugins ??= []
 
     // Force to SPA mode always since we don't need SSR for a desktop app.
     nuxt.options.ssr = false
 
     if (isProduction) {
-        // Fix path to make it works with Electron protocol `file://`
-        nuxt.options.app.baseURL = './'
-        nuxt.options.runtimeConfig.app.baseURL = './'
-        nuxt.options.router.options.hashMode = true // Avoid 404 errors
+      // Fix path to make it works with Electron protocol `file://`
+      nuxt.options.app.baseURL ??= './'
+      nuxt.options.runtimeConfig.app.baseURL ??= './'
+      nuxt.options.router.options.hashMode ??= true // Avoid 404 errors
```

Comment:
@gurvancampion 👋
1. Maybe user need to set `baseURL = './foo'`
2. Mabye user use custom protocol in Electron Main

So, we cannot force the default settings to be cahnged.